src/components/Block/ResultsError.vue

```diff
@@ -0,0 +1,15 @@
+<script setup>
+import { useI18n } from 'vue-i18n';
+const { t } = useI18n();
+</script>
+
+<template>
+  <Block :title="$t('results')" icon="warning" iconClass="text-red">
+    <div>{{ t('resultsError') }}</div>
+    <UiButton class="w-full mt-3">
```

Comment:
```suggestion
    <UiButton no-focus class="w-full mt-3">
```

Reply:
Why no focus? All the button should be clickable, with a > UIButton not UIButton > a

Reply:
We use the button for input too with focus-within class. On my Proposal creation PR I add a way to remove focus-within by for button, otherwise they are still highlighted after click.

Reply:
After click the button should be disabled